pkg/systemd/overview-cards/systemInformationCard.jsx

```diff
@@ -81,6 +92,43 @@ export class SystemInfomationCard extends React.Component {
                 });
     }
 
+    getSystemUptime() {
+        cockpit.spawn(["cat", "/proc/uptime"])
```

Comment:
Please use the [file API](https://cockpit-project.org/guide/latest/cockpit-file.html) instead of spawning a process, that's much more efficient. I. e. `cockpit.file("/proc/uptime").read()`.

Reply:
Changed to cockpit.file().read(). Is it mandatory to call close() at the end or is this done automatically at some point in time?

Reply:
It would be cleaner to do that, but AFAIK it's only important when using `.watch()`. Watching /proc/uptime is a no-go for several reasons, so that doesn't apply.